core/management/commands/purge_batch.py

```diff
@@ -29,6 +29,9 @@ def add_arguments(self, parser):
         parser.add_argument(
             '--optimize', action='store_true', default=False, dest='optimize',
             help='Optimize Solr and MySQL after purge (VERY SLOW)')
+        parser.add_argument('--interactive', action='store_true', default=True,
```

Comment:
Same issue as with the other `--interactive` flag: this can never be made `False`